Code CleanUp + Rename Namespace

Basicprogrammer10 opened the pull request:
I renamed your Namespace of `MEGClac` to `MEGCalc`. (lol)
I added a `.gitignore`
I also applied some automated code formatting / cleanup.

tycrek reviewed (commented):
Looks great. @Basicprogrammer10, I don't write C#, would you be able to explain the binary's that are included in the PR?

Basicprogrammer10 commented:
**Sure, I will explain in more depth what I have done.** 
(The Binary files are not supposed to be there lol... They are builds of the program but don't need to be uploaded to git like that. I have removed them)

1. **Git Ignore** - Just added this to so git doesn't upload any unnecessary files like IDE caches etc. It is mainly just from the c# git ignore template, but it has been slightly modified. This is the main cause of so many lines of code being added.
2.  **MEGClac.sln** - This file was removed because it is not needed in the for this project...
3.  **Form1.Designer.cs** - This file basically defines this applications GUI. Many changes have been made to this file, This is because I have corrected some spelling errors as well as moved some GUI elements to look a bit cleaner.
4.  **Form1.cs** - This is the main code for the GUI. The changes to this file are mainly renaming variables, Fixing spelling errors and making the code cleaner.
5.  **Program.cs** - This File contains the code that is run when the application starts (like a Main Function on C(++) or Rust etc.) it is used to start the Main GUI. In this file I changed the namespace name, removed some unused Imports and function private.
6.  **AssemblyInfo.cs** - Just removed an unneeded Import

**=== Removed Files ===**
7.  **MEG Calculator.exe** - This file was deleted because it is part of the build of the program, If you want this file for people to download and run it should be added under the releases section of the repo
8. **MEG Calculator.exe.config** - Same as #7
9. **MEG Calculator.pdb ** - Same as #7
10. **DesignTimeResolveAssemblyReferences.cache** - Same as #7
11. **DesignTimeResolveAssemblyReferencesInput.cache** - Same as #7
12. **MEG Calculator.exe** - Same as #7
13. **MEG Calculator.pdb** - Same as #7
14. **MEGCalc.Properties.Resources.resources** - Same as #7
15. **MEGClac.Form1.resources** - Same as #7
16. **MEGClac.csproj.CoreCompileInputs.cache** - Same as #7
17. **MEGClac.csproj.FileListAbsolute.txt** - Same as #7
18. **MEGClac.csproj.GenerateResource.cache** - Same as #7
19. **MEGClac.csprojAssemblyReference.cache** - Same as #7
20. **Properties.Resources.Designer.cs.dll** - Same as #7

I hope this helps clear up what my changes are about.

— Sigma : )

tycrek commented:
Thanks a ton for the summary @Basicprogrammer10! @Jonanen, these changes look good to me, feel free to merge if you're ready (or let me know if you want me to if you are unsure)
